### DISCUSSION TITLE
Add group preset for aws-sdk-client-mock monorepo

### DISCUSSION THREAD

**Discussion by WtfJoke** [renovatebot/renovate#28077] [Suggest an Idea]
### Tell us more.

Both dependencies, [aws-sdk-client-mock](https://www.npmjs.com/package/aws-sdk-client-mock) and [aws-sdk-client-mock-jest](https://www.npmjs.com/package/aws-sdk-client-mock-jest) are both published by the same github mono-repository: https://github.com/m-radzikowski/aws-sdk-client-mock and rely on each other and therefore should be upgraded together.

**Comment by secustor** [comment]
PRs welcome. 

See https://github.com/renovatebot/renovate/pull/27643
as example for this.

**Reply by WtfJoke** [reply]
Yeah I've already opened one yesterday :) https://github.com/renovatebot/renovate/pull/28078#pullrequestreview-1955458341

**Comment by WtfJoke** [comment]
Fixed with https://github.com/renovatebot/renovate/pull/28078
